### ISSUE TITLE
11/06/24_<NY><Android><Master> <Accessibility fixes > After the Ride is ended, "yes" / "No" are not announced as radio buttons/ selectable options (Parent id_1980)

### ISSUE THREAD

**Issue by Sumedh-14** [nammayatri/nammayatri#7257]
#1980 
[ After the Ride is ended, "yes" / "No" are not announced as radio buttons/ selectable options](https://drive.google.com/file/d/1rm4kn61kV3BEbiSQd5Qp1tkxQxmvBajm/view?usp=sharing)

**Steps To Reproduce**
1. Turn on Talkback > Launch the application and login with the valid credentials 
2. Book a Ride and once it is ended we will get "Has driver offered assistance.." texts
3. Slide through Yes / No options and observe 

**Expected Result**
It should be announced as buttons/ selectable options

**Actual Result**
It is announced as only "Yes" and "No". So user is not able to know whether it is an option to be selected.

**Comment by Shikhar-24** [comment]
expected as its a carousel view.
Selected option will be pronounced with done button.
